src/ceph_mon.cc

```diff
@@ -480,7 +480,7 @@ int main(int argc, const char **argv)
            << std::endl;
       exit(-err);
     }
-    if (stats.avail_percent <= g_conf->mon_data_avail_crit) {
+    if (static_cast<uint32_t>(stats.avail_percent) <= g_conf->mon_data_avail_crit) {
```

Comment:
@oliveiradan Please keep PR title < 50 characters
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits